internal/app/subsystems/aio/network/network_dst.go

```diff
@@ -69,10 +70,18 @@ func (d *NetworkDSTDevice) Process(sqes []*bus.SQE[t_aio.Submission, t_aio.Compl
 				res = &http.Response{
 					StatusCode: http.StatusOK,
 				}
+
+				event := announcements.NewEvent("HTTPResponse")
```

Comment:
We need to announce more information about the HttpRequest e.g. url, method, etc to be able to route and assert in model